curriculum/challenges/english/10-coding-interview-prep/rosetta-code/count-the-coins.md

```diff
@@ -30,7 +30,7 @@ There are four types of common coins in [US](https://en.wikipedia.org/wiki/Unite
 
 # --instructions--
 
-Implement a function to determine how many ways there are to make change for a dollar using these common coins (1 dollar = 100 cents)
+Implement a function to determine how many ways there are to make change for a `cents` cents using these common coins.
```

Comment:
Not sure I understand this change from "dollar" to "`cents` cents`".

Reply:
I figured it will be more convenient to use certain number of cents as a target and keep as the highest requested value still one dollar, just in cents. Instead of dealing with fractions when adding some intermediate tests for amounts lower than one dollar.

Reply:
I think we should stick as close to the original Rosetta Code challenge as possible.

Reply:
I'd say this change isn't making it further from the original, than if dollar would be used as a quantity. Considering that original challenge doesn't give explicit input value, somebody solving it, could take the later mentioned 100 cents as input from the start.